parts/kubernetesmasteraddons-tiller-deployment.yaml

```diff
@@ -0,0 +1,117 @@
+# Produced by 'helm init --dry-run --debug'
+apiVersion: extensions/v1beta1
```

Comment:
Can we please clean this manifest up given that most of it is null. It also needs the following label for addonmanager to pick it up

```
  labels:
    kubernetes.io/cluster-service: "true"
```